src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/ConnectionTestParametersData.cs

```diff
@@ -46,6 +46,7 @@ public ConnectionTestParametersData()
         {
             ConnectionTestParametersList = new List<ConnectionTestParameters>
             {
+                // 1
```

Comment:
please change them to be more meaningful comments. What are each number related to. This makes the code reading easier.